nifi-nar-bundles/nifi-dropbox-bundle/nifi-dropbox-processors/src/test/java/org/apache/nifi/processors/dropbox/ListDropboxTest.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.processors.dropbox;
+
+import static org.apache.nifi.util.EqualsWrapper.wrapList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Answers.RETURNS_DEEP_STUBS;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.dropbox.core.v2.DbxClientV2;
+import com.dropbox.core.v2.files.DbxUserFilesRequests;
+import com.dropbox.core.v2.files.FileMetadata;
+import com.dropbox.core.v2.files.ListFolderBuilder;
+import com.dropbox.core.v2.files.ListFolderResult;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.function.Function;
+import org.apache.nifi.processor.ProcessContext;
+import org.apache.nifi.util.EqualsWrapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ListDropboxTest {
+
+    private ListDropbox testSubject;
+
+    private ProcessContext mockProcessContext;
+    private DbxClientV2 mockDbxClient;
+    private DbxUserFilesRequests mockDbxUserFilesRequest;
+    private ListFolderResult mockListFolderResult;
+    private ListFolderBuilder mockListFolderBuilder;
+
+
+    @BeforeEach
+    void setUp() {
+        mockProcessContext = mock(ProcessContext.class, RETURNS_DEEP_STUBS);
+        mockDbxClient = mock(DbxClientV2.class, RETURNS_DEEP_STUBS);
+        mockDbxUserFilesRequest = mock(DbxUserFilesRequests.class, RETURNS_DEEP_STUBS);
+        mockListFolderResult = mock(ListFolderResult.class, RETURNS_DEEP_STUBS);
+        mockListFolderBuilder = mock(ListFolderBuilder.class, RETURNS_DEEP_STUBS);
```

Comment:
Instead of using the static `mock()` call, recommend annotating the class with `ExtendWith(MockitoExtension.class)` and then annotating the mock objects with `@Mock`. That allows Mockito to instantiate the mock objects without this method.